packages/synapse-react-client/src/components/styled/StyledTableContainer.tsx

```diff
@@ -12,6 +13,8 @@ export const StyledTableContainer: StyledComponent<StyledTableContainerProps> =
     th: {
       height: '38px',
       backgroundColor: theme.palette.grey[200],
+      borderRight: '1px solid',
```

Comment:
Moved many styles from CSS applied to only `SynapseTable` to this shared styled component.